engine/src/test/resources/log4j.properties

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-log4j.rootLogger=INFO,CONSOLE
+log4j.rootLogger=INFO,CONSOLE,malhar
```

Comment:
Yes, it adds extra logger, which should not break anything, this is extra logger. I will still try to see if I can use custom properties file for our particular test.